### DISCUSSION TITLE
FEATURE REQUEST: setting to disable right click menu

### DISCUSSION THREAD

**Discussion by dimateos** [furybee/chrome-tab-modifier#277] [Ideas]
Hey!

I was looking for an option to toggle off and remove the extension from the right click menu:

* I never use that feature and would like to declutter the right click menu
* Other extensions have a setting to do this e.g. ublock

Thank for the extension!

**Comment by sylouuu** [comment]
Duplicate of #206

**Reply by ghost** [reply]
The option to hide it in the right click menu can be a useful feature, please add it

**Comment by dimateos** [comment]
AFAIK all issues were marked as duplicated + completed SO none was actually left open to track the progress of this feature request.
May I open a new one or you could reopen one of those. thanks!
